analyzers/rspec/cs/S3363.json

```diff
@@ -0,0 +1,15 @@
+{
+  "title": "Timestamps should not be used as primary keys",
```

Comment:
WDYT?
```suggestion
  "title": "Temporal types should not be used as primary keys",
```

Reply:
I used that earlier, but temporal types [has its own meaning in SQL and Entity Framework](https://www.learnentityframeworkcore.com/misc/temporal-tables), so I dropped it to avoid confusion. Chronological types sounded a little weird, so I ended up using Timestamp. It's a stretch (considering that I'm about to add the `DateOnly` type as well), but not a long stretch.

Reply:
Hmm, we can in that case either leave as is or use the Date-time version. That might be good. I leave this with you to decide. No strong opinion on it.